modules/ebi-metagenomics/catcontigs/meta.yml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "CATCONTIGS"
+description: MGnify contig taxonomy assignation using CAT"
+keywords:
+  - mags
+  - taxonomy
+  - genomics
+tools:
+  - "catcontigs":
+      description: "CAT/BAT: tool for taxonomic classification of contigs and metagenome-assembled
+        genomes (MAGs)"
+      homepage: "https://github.com/MGXlab/CAT_pack"
+      documentation: "https://github.com/MGXlab/CAT_pack"
+      tool_dev_url: "https://github.com/MGXlab/CAT_pack"
+      doi: "10.1038/s41467-024-47155-1."
+      licence: ["MIT"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - contigs_fasta:
+        type: file
+        description: The fasta file that contains contigs
+        pattern: "*.{fa,fasta}"
+  - - cat_db:
+        type: directory
+        description: Path to directory that contains CAT database files
+  - - taxonomy_db:
```

Comment:
from your explanation, shouldn't these have only one dash then? since they are:
```
    path cat_db
    path taxonomy_db
```
as inputs, no tuple

Reply:
well... the linter doesn't like that:
```
The meta.yml of the module catcontigs is not valid: {'cat_db': {'type': 'directory', 'description': 'Path to directory that contains CAT database files'}} is not of type 'array'. Check the entry for input.
```

:sigh: I leave this one as it is, nf-core + nextflow are working on this (trying to use the schemas directly in nextflow) so let's wait